src/Flex.php

```diff
@@ -39,13 +37,41 @@
  */
 class Flex implements PluginInterface, EventSubscriberInterface
 {
+    /**
+     * @var Composer
+     */
     private $composer;
+
+    /**
+     * @var IOInterface
+     */
     private $io;
+
+    /**
+     * @var Options
+     */
     private $options;
+
+    /**
+     * @var Configurator
+     */
     private $configurator;
+
+    /**
+     * @var Downloader
+     */
     private $downloader;
+
+    /**
+     * @var array
+     */
     private $postInstallOutput = [''];
+
+    /**
+     * @var \Closure
+     */
```

Comment:
Not a big fan of documenting private stuff. If others think this is useful (I don't), can we at least put all `@var` blocks on one line?